### DISCUSSION TITLE
My github copilot subscription ended in 11 days.

### DISCUSSION THREAD

**Discussion by AnibalDBXD** [community/community#39069] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I purchased GitHub copilot on November 3rd and now it says "Thank you for using GitHub Copilot. Your subscription has ended"

I have the invoice and it's only been 11 days since I paid :/

**Comment by iamcjwilliams** [comment]
Hello @AnibalDBXD We're sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
